src/lib.rs

```diff
@@ -109,3 +109,17 @@ pub fn start_server() {
         }
     };
 }
+
+/// Record the audio output of a Sonic Pi session to a local file.
+/// Stop and save the recording when the <Enter> key is pressed.
+///
+pub fn record(path: &str) {
+    server::start_recording();
+    let mut input = String::new();
```

Comment:
Could we print a message here instructing the user to press enter to stop recording? :)